.github/CODEOWNERS

```diff
@@ -112,7 +112,7 @@
 /cmd/cluster-agent/                     @DataDog/container-integrations
 /cmd/cluster-agent/commands/            @DataDog/container-integrations @DataDog/integrations-tools-and-libraries
```

Comment:
not sure about this one

Reply:
I don't think we own anything on that folder, it's probably outdated.